src/libstore/build.cc

```diff
@@ -2439,7 +2471,7 @@ void DerivationGoal::runChild()
 
             /* Our ancestry. N.B: this uses literal on folders, instead of subpath. Without that,
                you open up the entire filesystem because you end up with (subpath "/") */
-            sandboxProfile += "(allow file-read-metadata\n";
+            sandboxProfile += "(allow file-read*\n";
```

Comment:
Yes, but since this makes canonicalizePath work and the builder still can't *read* files outside of the build tree, I didn't think it was harmful.